src/host/ft_uia/KeyPressTests.cs

```diff
@@ -208,7 +208,7 @@ public void VerifyCtrlCCmd()
                         for (int i = 0; i < text.Count(); ++i)
                         {
                             string line = text.ElementAt(i);
-                            line.Trim(' ');
+                            line = line.Trim(' ');

```

Comment:
does the test still pass with this change?

Reply:
yes